bin/www

```diff
@@ -0,0 +1,8 @@
+const app = require('../index.js');
+const { port } = require('../config');
+
+const connectionPort = port || 3000;
```

Comment:
`port || 3000`! Defaulting to `3000` could take place in the config file since there's one.

Reply:
resolved